packages/core/directives/marker.ts

```diff
@@ -214,6 +224,20 @@ export class AgmMarker implements OnDestroy, OnChanges, AfterContentInit {
             });
     this._observableSubscriptions.push(ds);
 
+    const dt =
```

Comment:
could you refactor these 3 events to a single loop?: dragend, dragstart and drag
Because the code for these 3 is exactly the same.